tests/e2e/specs/out-of-game/lobby.spec.js

```diff
@@ -245,112 +304,263 @@ describe('Lobby - P1 Perspective', () => {
+function checkPageContentOfLobby() {
+  // Displays title, game name and nav drawer
+    cy.contains('h1', en.lobby.lobbyFor);
+    cy.window()
+      .its('cuttle.gameStore')
+      .then((gameData) => {
+        cy.contains('h5', gameData.name);
+      });
+    cy.get('[data-cy=nav-drawer]').should('not.exist');
+  // Shows exit button, ready buttonn casual/ranked switch
+    cy.contains('button.v-btn', en.lobby.exit);
+    cy.contains('button.v-btn', en.lobby.ready);
+    cy.get('[data-cy=edit-game-ranked-switch]').should('exist');
+}
+
+function checkRanked(isRanked) {
+  if(isRanked) {
+    cy.get('[data-cy=ready-button-coffee-icon]').should('not.exist');
+    cy.get('[data-cy=ready-button-sword-cross-icon]').should('exist');
+    cy.contains('[data-cy=edit-game-ranked-switch]', en.global.ranked);
+  } else {
+    cy.get('[data-cy=ready-button-coffee-icon]').should('exist');
+    cy.get('[data-cy=ready-button-sword-cross-icon]').should('not.exist');
+    cy.contains('[data-cy=edit-game-ranked-switch]', en.global.casual);
+  }
+
+  cy.window()
+  .its('cuttle.gameStore')
+    .then((game) => {
+      expect(game.isRanked).to.eq(isRanked);
+    });
+}
+
+function assertGameStarted(noMovesYet = true) {
+  cy.url().should('include', '/game');
+  cy.window()
+    .its('cuttle.gameStore')
+    .then((game) => {
+      expect(game.players[1].hand.length).to.eq(6);
+      if (noMovesYet) {
+        expect(game.players.length).to.eq(2);
+        expect(game.players[0].hand.length).to.eq(5);
+        expect(game.deck.length).to.eq(39);
+      }
+      expect(game.topCard.rank).to.be.greaterThan(0);
+      expect(game.secondCard.rank).to.be.greaterThan(0);
+      expect(game.scrap.length).to.eq(0);
+      expect(game.twos.length).to.eq(0);
+    });
+}
+
+function clickReadyButton() {
+  cy.get('[data-cy=ready-button]').click();
+}
+
+function testReadyValueInState(player, value) {
+  let prop = player === 'p0' ? 'p0Ready' : 'p1Ready';
+  cy.window()
+    .its('cuttle.gameStore')
+    .then((game) => {
+      expect(game[prop]).to.eq(value);
+    });
+}
+function checkOpponentIsReadyInGetter(status) {
+  cy.window()
+  .its('cuttle.gameStore')
+  .then((store) => {
+    expect(store.opponentIsReady).to.eq(status);
+  });
+
+}
+
+
+function indicatorShowsNameAndCard(user) {
+  const userString = user === 'player' ? 'my' : 'opponent';
+
+  if(user === 'player') {
+    cy.get(`[data-cy=${userString}-indicator]`).contains(myUser.username);
+  } else {
+    cy.get(`[data-cy=${userString}-indicator]`).contains(opponentOne.username);
+  }
+
+  cy.get(`[data-cy=${userString}-indicator]`).find('[data-cy="lobby-card-container"]')
+    .should('exist');
+  cy.get(`[data-cy=${userString}-indicator]`).find('[data-cy="lobby-back-card"]').should('exist');
+  cy.get(`[data-cy=${userString}-indicator]`).find('[data-cy="lobby-ready-card"]').should('exist');
+}
+
+function opponentIndicatorShouldBeEmpty() {
+  cy.get('[data-cy=opponent-indicator]').contains('Invite');
+}
```

Comment:
This as well, I think we should just use the regular cypress command

Reply:
I feel it would be more consistent with the other similar functions.  It seems more clear what the test is. 
But I don't mind swapping it back.
As I was rewriting the tests, it was much easier to read function names, that's why there all over the place.

Reply:
While I agree that it's clearer this way, I'm not sure it's worth another function for a single assertion, especially the way cuttle is setup IMO the assertion is pretty clear.

Reply:
It’s not worth the abstraction, continue using cypress commands directly until there’s more to encapsulate.